### ISSUE TITLE
Screenshots

### ISSUE THREAD

**Issue by fulldecent** [webmaster442/macrokeypad#1]
Hello, please include screenshots of your interface and hopefully some of the keyboards you support.

I think you are targeting this software:

<img width="711" alt="Screenshot 2022-11-18 at 09 54 28" src="https://user-images.githubusercontent.com/382183/202733891-4eee4225-0ef1-4722-9d30-c99589a18345.png">

Example keyboards:

<img width="844" alt="Screenshot 2022-11-18 at 09 59 24" src="https://user-images.githubusercontent.com/382183/202734357-a096bb59-0c1a-407d-b049-c1fe9588f8d1.png">

Here is the first-party application from a big seller on AliExpress. I got write access, when we are ready I can link to this repo from that folder.

https://drive.google.com/drive/folders/1ELKgQ2BUPMGbhAVZO2o962LVMWzFgmp3

**Comment by webmaster442** [comment]
Yes, The App on the screenshot is the one that I reverse engineered. I Can add a screenshot of it of course, but I can't include screenshots that I support, because I only have one of these and I'm not the official developer. I just reverse engineered the stuff and ported it to a more modern .NET version. However I'm open to accept pull requests for this repository. So If you happen to have images and model numbers of keyboards that the app supports, feel free to create a merge request.
